### PR TITLE
fixing auto-discovery URL for `linka_aar_bike`

### PR DESCRIPTION
**Pull request by drewda**
Please review and consider adding this fix to one of the recently added GBFS auto-discovery URLs.

We noticed this when it broke our down-stream script that ingests a portion of the fields from systems.csv: https://github.com/transitland/transitland-atlas/commit/2c11b474bcacedd51b005d7c7008991d346ddbf5#diff-839c6874cb906b6656180f2fd649476ba38ca60b0db9e67cc05c72eec224f18eR8

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -644,7 +644,7 @@ SI,Nomago Bikes - SLOVENIA,"Kranjska Gora, SI",nextbike_cx,https://bikes.nomago.
...
 SK,Arriva Nitra Slovakia,"Nitra, SK",nextbike_as,https://arriva.bike/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_as/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_as%2Fgbfs.json
 SK,BikeKIA,"Žilina, SK",nextbike_ak,https://bikekia.sk/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ak/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ak%2Fgbfs.json
-TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/…,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2F%E2%80%A6
+TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2F%E2%80%A6
```
Comment (richfab) [inline systems.csv]: Hello @drewda, 
Great catch! Thank you very much for this contribution.
Let's please update the validation report URL too.
Thank you!

```suggestion
TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2Fgbfs.json
```

**Review by richfab — CHANGES_REQUESTED** [review]
Thank you @drewda!
I left a suggestion. LGTM otherwise!

**Comment by drewda** [comment]
@richfab thanks for reviewing!

**Review by richfab — APPROVED** [review]
LGTM, thank you!
Merging now 🚀

**Pull request merged**